tests/foreman/api/test_host.py

```diff
@@ -22,7 +22,11 @@
 from fauxfactory import gen_integer, gen_ipaddr, gen_mac, gen_string
 from nailgun import client, entities
 from requests.exceptions import HTTPError
+from six.moves import http_client
```

Comment:
Isn't `six` a third-party module?
And according to http://robottelo.readthedocs.io/en/latest/code_standards.html#recommended
`third-party` should be placed before `application-specific` and separated with `a blank line between groups of imports`.